pkg/chunk/local/boltdb_table_client.go

```diff
@@ -36,6 +36,9 @@ func (c *TableClient) ListTables(ctx context.Context) ([]string, error) {
 }
 
 func (c *TableClient) CreateTable(ctx context.Context, desc chunk.TableDesc) error {
+	if _, err := os.OpenFile(filepath.Join(c.directory, desc.Name), os.O_RDWR|os.O_CREATE, 0666); err != nil {
+		return err
+	}
```

Comment:
I think this would keep the file reference open. Let us do this instead.
```suggestion
	return ioutil.WriteFile(filepath.Join(c.directory, desc.Name), nil, 0666)
```

Reply:
Perfect. Thanks @sandeepsukhani 
Other way we can call `Close()` on the file. However, let's stick to your code. Less number of lines 😄